apps/monitoring/src/ui/datadog/datadog-config.tsx

```diff
@@ -175,114 +143,111 @@ export const DatadogConfig = () => {
-        <div>
-          <Controller
-            render={({ field }) => {
-              return (
-                <FormGroup row>
-                  <FormControlLabel
-                    control={<Checkbox checked={field.value} {...field} />}
-                    label="Enabled"
-                  />
-                </FormGroup>
-              );
-            }}
-            name="active"
-            control={control}
-          />
-          <InputLabel id="datadog-site-label">Datadog Site</InputLabel>
-          <Select
-            defaultValue={DatadogSite.Us1}
-            fullWidth
-            labelId="datadog-site-label"
-            {...register("site")}
-          >
-            {Object.values(DatadogSite).map((v) => (
-              <MenuItem value={v} key={v}>
-                {v}
-              </MenuItem>
-            ))}
-          </Select>
-        </div>
-        <TextField
-          fullWidth
-          variant="standard"
-          label="Api Key"
-          defaultValue=""
-          helperText={<ApiKeyHelperText site={activeSite} />}
-          {...register("apiKey")}
-        />
-        {queryData.data?.integrations.datadog?.error && (
-          <Typography color="error">{queryData.data?.integrations.datadog?.error}</Typography>
-        )}
-        <Button type="submit" variant="primary" fullWidth>
-          Save configuration
-        </Button>
-        <Button
-          type="reset"
-          variant="secondary"
-          fullWidth
-          onClick={(e) => {
-            e.preventDefault();
-
-            deleteCredentials({}).then(() => {
-              fetchConfig();
-              reset();
-              notifySuccess("Configuration updated", "Successfully deleted Datadog settings");
+              if (errors?.length) {
+                notifyError("Error configuring Datadog", errors[0].message);
+              }
             });
-          }}
+          })}
         >
-          Delete configuration
-        </Button>
-      </form>
-    </Section>
+          <Box as={"label"} display={"flex"} gap={2} marginBottom={4}>
+            <Toggle control={control} name={"active"} />
+            <Text variant={"bodyEmp"}>Active</Text>
+          </Box>
+
+          <Select
+            label={"Datadog Site"}
+            options={Object.values(DatadogSite).map((v) => ({
+              label: v,
+              value: v,
+            }))}
+            control={control}
+            name={"site"}
+          />
+          <Box marginY={4}>
+            <Input
+              label="Api Key"
+              defaultValue=""
+              helperText={<ApiKeyHelperText site={activeSite} />}
+              control={control}
+              name={"apiKey"}
+            />
+          </Box>
+          {queryData.data?.integrations.datadog?.error && (
+            <Text color={"textCriticalDefault"}>{queryData.data?.integrations.datadog?.error}</Text>
+          )}
+          <Box display={"flex"} gap={2} marginTop={8}>
```

Comment:
On other apps we have buttons aligned to right hand side.
Probably would also move non destructive (Save) button to the right to minimize accidental clicks.